app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -191,6 +197,25 @@ abstract class AppDatabase : RoomDatabase() {
       }
     }
   }
+
+  class Migration_8_9 : Migration(8, 9) {
```

Comment:
@pratul, @sanchita and I had a discussion yesterday about placing each migration in a separate file. Can you do this as part of this PR?

Reply:
Should we then also keep the test for every migration in a different class?

Reply:
Not necessary at this point, I feel. Maybe in the future.

Let's drop this change from this PR for now, and do it in a  separate one.

Reply:
Aye